Makefile

```diff
@@ -31,9 +31,11 @@ endif
 
 APPNAME    = Zilliqa
 ifeq ($(TARGET_NAME), TARGET_NANOS)
-ICONNAME   = zilliqa-nano-s.gif
+ICONNAME   = icons/zilliqa_nanos.gif
+else ifeq ($(TARGET_NAME),TARGET_STAX)
+ICONNAME   = icons/zilliqa_stax_32px.gif
 else
-ICONNAME   = zilliqa-nano-x.gif
+ICONNAME   = icons/zilliqa_nanox.gif
```

Comment:
Could we please update these? 
![Zilliqa horizontal white black-03](https://user-images.githubusercontent.com/574350/221793758-2f90092d-6d01-4be6-a8f7-50d63ab1cab3.svg)
![Zilliqa horizontal white black-04](https://user-images.githubusercontent.com/574350/221793773-c10b2913-afa6-44ff-a64a-574e7b9be305.svg)

Reply:
I can update the logo but I need 32*32pixels and 64*64pixels versions with some margins following the guidelines I shared on Discord.

Reply:
Perfect. I'll have a look and get back to you